src/run-block-actions.ts

```diff
@@ -26,16 +19,13 @@ export const RunBlockAction = async (
     );
   }
 
+  const blockActionBody = baseHandlerArgs.body as BlockActionInvocationBody;
   // We don't catch any errors the handler may throw, we let them throw, and stop the process
   // deno-lint-ignore no-explicit-any
   const response: any = await handler({
-    inputs,
-    env,
-    token,
-    team_id,
-    enterprise_id,
-    body,
-    action: body.actions[0],
+    ...baseHandlerArgs,
+    action: blockActionBody.actions[0],
```

Comment:
The thing that appeals to me in this refactor is that it is much clearer which userland handler arguments are unique or specific to each event just by looking at how the userland `handler` is invoked.